website/config/routes.js

```diff
@@ -26,15 +26,6 @@ module.exports.routes = {
     }
   },
 
-  'GET /get-started': {
-    action: 'view-get-started' ,
-    locals: {
-      currentPage: 'get started',
-      pageTitleForMeta: 'Get started | Fleet for osquery',
-      pageDescriptionForMeta: 'Learn about getting started with Fleet.'
-    }
-  },
-
```

Comment:
To maintain backwards compatibility of links, let's replace this with a redirect down in the redirects section.  (To where?  Homepage is probably the best bet.)

Reply:
@eashaw Nope, all good!  I just missed the redirect somehow.